sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -290,6 +292,35 @@ The returned response is a heterogeneous list of result and error objects: list[
 Please refer to the service documentation for a conceptual discussion of [entity linking][linked_entity_recognition]
 and [supported types][linked_entities_categories].
 
+### Recognize PII entities
+[recognize_pii_entities][recognize_pii_entities] recognizes and categorizes Personally Identifiable Information (PII) entities in its input text, such as
```

Comment:
there's no link defined for `recognize_pii_entities`

Reply:
thanks!